services/comments.js

```diff
@@ -28,3 +28,20 @@ exports.createComment = async ({ userId, newsId, body }) => {
     throw new ErrorObject(err.message, 500)
   }
 }
+
+exports.deleteComment = async (id, user) => {
+  try {
+    const comment = await Comment.findByPk(id)
+
+    if (!comment) {
+      throw new ErrorObject('Comment not found', 404)
+    }
+    if (comment.userId !== user.id) {
+      throw new ErrorObject('Invalid login credentials', 401)
```

Comment:
pondria otro mensage y otro status code
en vez de un 401 pondria un 403. 401 se usa cuando no hay token o no es valido. en este caso, el token si es valido (lo comprueban los middlewares) pero el usuario tiene permisos insuficientes, que es para lo que es usa el 403
tambien por eso cambiaria el mesaje porque no es lo que realmente esta pasando. podrias poner algo como `'You don't have enough permissions to delete this comment'`

https://auth0.com/blog/forbidden-unauthorized-http-status-codes/#Recap

Reply:
Quise poner el mismo mensaje de auth a proposito! pero ya lo cambio